modules/lemmaBidAdapter.js

```diff
@@ -368,11 +376,9 @@ function _getImpressionObject(bid) {
     secure: window.location.protocol === 'https:' ? 1 : 0,
     bidfloorcur: params.currency ? params.currency : DEFAULT_CURRENCY
   };
-
-  if (params.bidFloor) {
-    impression.bidfloor = params.bidFloor;
-  }
-
+  
+  _setFloor(impression, bid);
+  
```

Comment:
You have lint error here.  
Check output of circle ci, or just run `gulp lint` on your local to fix this error.

Reply:
I have run gulp lint and there is not showing any error on the terminal.

Reply:
It won't show you any error, it will automatically fix it.  
You have trailing spaces on lines 379 and 381, see circle ci output.

Reply:
Remove trailing spaces on lines 379 and 381.